profiler/CMakeLists.txt

```diff
@@ -88,6 +88,27 @@ add_definitions("-D_GLIBCXX_USE_CXX11_ABI=0")
 # Output folders
 # ******************************************************
 
+# Deployment directory
+
+if (ISAMD64)
+    SET(ARCH_POSTFIX "x64")
+elseif (ISARM64)
+    SET(ARCH_POSTFIX "arm64")
+elseif (ISARM)
+    SET(ARCH_POSTFIX "arm")
+else()
+    SET(ARCH_POSTFIX "x86")
+endif()
+
+SET(ARCH_BASENAME "linux")
+if (DEFINED ENV{IsAlpine} AND "$ENV{IsAlpine}" MATCHES "true")
+    SET(ARCH_BASENAME "${ARCH_BASENAME}-musl")
+else()
+    SET(ARCH_BASENAME ${ARCH_BASENAME})
```

Comment:
Why do you need this else since the value does not change?

Reply:
good one